src/gui/QvisTimeSliderControlWidget.C

```diff
@@ -266,22 +275,38 @@ QvisTimeSliderControlWidget::GetTimeStateFormat() const
 //   Brad Whitlock, Sun Jan 25 01:28:23 PDT 2004
 //   I made it use windowInfo instead of globalAtts.
 //
+//   Kathleen Biagas, Mon Sep 30 09:29:50 PDT 2019
+//   Added plotList. Set 'shouldEnablePlay; based on active, completed
+//   time-varying plot that follows time.
+//
 // ****************************************************************************
 
 void
 QvisTimeSliderControlWidget::Update(Subject *TheChangedSubject)
 {
-    if(fileServer == 0 || windowInfo == 0)
+    if(fileServer == 0 || windowInfo == 0 || plotList == 0)
         return;
     else if(TheChangedSubject == windowInfo)
         UpdateAnimationControls(false);
+    else if(TheChangedSubject == plotList)
+    {
+        bool shouldEnablePlay = false;
```

Comment:
This code block for iterating over the plot list to determine whether the plot is such that VCR buttons should be enabled is duplicated here and in `QvisTimeSliderControlWidget::ConnectPlotList`. Maybe re-factor to a static function in the file into which `plotList` is based and returns a `bool`?